tests/Testcontainers.Ollama.Tests/OllamaContainerTests.cs

```diff
@@ -0,0 +1,49 @@
+namespace Testcontainers.Ollama.Tests
+{
+  public class OllamaContainerTests : IAsyncLifetime
+  {
+    private readonly ITestOutputHelper _testOutputHelper;
+    private OllamaContainer _ollamaContainer;
+
+    public OllamaContainerTests(ITestOutputHelper testOutputHelper)
+    {
+      _testOutputHelper = testOutputHelper;
+    }
+
+    public async Task InitializeAsync()
+    {
+      TestcontainersSettings.Logger = new TestOutputLogger(nameof(OllamaContainerTests), _testOutputHelper);
```

Comment:
Can we please keep the default logger configuration, etc. I am aware of the issues around xUnit.net's `ITestOutputHelper` and console logging, but I would prefer to keep the modules and tests consistent. This helps in applying future refactoring, etc.

Reply:
I was a bit careless, and shouldn't have included it, its very much a "I like this to as a debug tool while working" so, removed